src/javaRosa.js

```diff
@@ -576,7 +576,12 @@ define([
 
         widget.save = function () {
             // override save to call out to rename itext
-            widget.mug.setItextId(widget.path, widget.getValue());
+            if (widget.mug.setItextId(widget.path, widget.getValue())) {
```

Comment:
Is it kosher in vellum to test the return value of a function in the if statement like this, or is that a dirty C habit I've picked?